src/test/operate.test.js

```diff
@@ -0,0 +1,36 @@
+import operate from '../logic/operate';
+
+it('plus', () => {
+  const result = operate(12, 12, '+');
+  expect(result).toBe('24');
+});
+
+it('minus', () => {
+  const result = operate(12, 12, '-');
+  expect(result).toBe('0');
+});
+
+it('times', () => {
+  const result = operate(12, 12, 'x');
+  expect(result).toBe('144');
+});
+
+it('div', () => {
+  const result = operate(12, 12, '÷');
+  expect(result).toBe('1');
+});
+
+it('div', () => {
+  const result = operate(12, 0, '÷');
+  expect(result).toBe('Can\'t divide by 0.');
+});
+
+it('mod', () => {
+  const result = operate(12, 12, '%');
+  expect(result).toBe('0');
+});
+
+it('mod', () => {
+  const result = operate(12, 0, '%');
+  expect(result).toBe('Can\'t find modulo as can\'t divide by 0.');
```

Comment:
- [ ] Same here, It would be better professionally if you include all test cases that are related to the same topic or functionality under a `describe` block.
[Hint]: See that [video](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjpqMnXorL4AhWmiv0HHabkA3wQFnoECAQQAw&url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DkVzw_f7TfCE&usg=AOvVaw0RUJ139efStEZZVJJ50P1z&cshid=1655392824985935) for more info about that.

Reply:
Thank you @Omar-Muhamad I have already added a description.